internal/clients/logger/github_logger/github_logger.go

```diff
@@ -15,19 +16,21 @@ const (
 
 // GithubLogger logger for Github Actions
 type GithubLogger struct {
-	log *log.Logger
+	log   *log.Logger
+	debug bool
```

Comment:
why does the logger care about this flag - is it so we can only write debug logs if it's true?

Reply:
I was not exactly sure how the debug flag would be used in Github Actions. I added to the GithubLogger with the thought that if it was true it would print the debug logs. But with Github Actions you still need to set the secret to make debug logs appear.

Reply:
yeah exactly. my thought is to not add this field to the GithubLogger since we don't explicitly need it, and then we can just use this flag client (and debug flag) for future iterations

Reply:
we should kill this field right?